### DISCUSSION TITLE
How to install vscode-cpptools

### DISCUSSION THREAD

**Discussion by rzksobhy78** [mfussenegger/nvim-dap#360] [Q&A]
I downloads this version
https://github.com/microsoft/vscode-cpptools/releases/download/1.7.1/cpptools-linux.vsix

but my OpenDebugAD7 is not executable
and Can't find mono.<system-<platform>

note: OpenDebugAD7 is not on extension/debugAdapters/OpenDebugAD7
its on extension/debugAdapters/bin/OpenDebugAD7

any Help ?

OS: Arch linux

**Comment by bew** [comment]
> but my OpenDebugAD7 is not executable

Make it executable?

And about mono, these recent PRs descriptions seems to say that they're not using mono anymore:
- https://github.com/microsoft/vscode-cpptools/pull/8046
- https://github.com/microsoft/vscode-cpptools/pull/7858

I'd say, try without bothering about it?

**Reply by rzksobhy78** [reply]
I just switched to linux how to do that

**Reply by bew** [reply]
What exactly do you not know how to do?

If it's making a file executable...
This is linux, learn to search on google and/or read the manuals!
Here is a hint: https://www.google.com/search?q=how+to+make+a+file+executable+in+linux
And/or another hint: the linux command for changing file properties is `chmod`, read the manual with `man chmod`
